<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutPage</class>
 <widget class="QWidget" name="AboutPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">About Page</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="logoImage">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../res/resources.qrc">:/icons/icons/icon.ico</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="appName">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>App name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="appDescription">
     <property name="text">
      <string>Description</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="copyrightMessage">
     <property name="text">
      <string>Copyright Message</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="technicalDetail">
     <property name="text">
      <string>Technical Detail</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="creditsButton">
       <property name="text">
        <string>Credits</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="licenseButton">
       <property name="text">
        <string>License</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="aboutQt">
       <property name="text">
        <string>About Qt</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>creditsButton</sender>
   <signal>clicked()</signal>
   <receiver>AboutPage</receiver>
   <slot>showContributorsPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>licenseButton</sender>
   <signal>clicked()</signal>
   <receiver>AboutPage</receiver>
   <slot>showLicensePage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>AboutPage</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aboutQt</sender>
   <signal>clicked()</signal>
   <receiver>AboutPage</receiver>
   <slot>showAboutQt()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showLicensePage()</slot>
  <slot>showContributorsPage()</slot>
  <slot>showAboutQt()</slot>
 </slots>
</ui>
